app/assets/javascripts/admin/addon/routes/admin.js

```diff
@@ -24,6 +26,12 @@ export default class AdminRoute extends DiscourseRoute {
     this.controllerFor("application").setProperties({
       showTop: false,
     });
+
+    this.store.findAll("plugin").then((plugins) => {
```

Comment:
OK so rather than loading the plugins with AJAX when /admin route is opened, we can instead just change our preloaded data for admin. I previously added this `enabledPluginAdminRoutes` for the sidebar:

https://github.com/discourse/discourse/blob/d90dd6470c7837c1be87de245132b68eafbef14c/app/controllers/application_controller.rb#L675-L678

Let's just change it to `store_preloaded("enabledPlugins")` and use `Discourse.plugins_sorted_by_name`, then change how admin-sidebar JS loads the `enabledPluginAdminRoutes`. We can then use `PreloadStore.get("enabledPlugins")` here and here:

https://github.com/discourse/discourse/blob/d90dd6470c7837c1be87de245132b68eafbef14c/app/assets/javascripts/admin/addon/routes/admin-plugins.js#L7-L12